praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/rest/PraktikumsstellenController.java

```diff
@@ -81,7 +81,7 @@ public TreeMap<String, List<PraktikumsstelleDto>> getAllPraktiumsstellenInSpecif
     @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
     @PatchMapping("/{praktikumsstellenId}")
     @ResponseStatus(HttpStatus.OK)
-    public PraktikumsstelleDto assignNwk(@PathVariable UUID praktikumsstellenId, @RequestParam(required = false) UUID nwkId) {
+    public PraktikumsstelleDto assignNwk(@PathVariable UUID praktikumsstellenId, @RequestParam(name = "nwkId", required = false) UUID nwkId) {
```

Comment:
```suggestion
    public PraktikumsstelleDto assignNwk(@PathVariable(name = "praktikumsstellenId") UUID praktikumsstellenId,
            @RequestParam(name = "nwkId", required = false) UUID nwkId) {
```
Seems like this is needed to fix it completely